src/common/drpc.c

```diff
@@ -695,7 +646,43 @@ drpc_close(struct drpc *ctx)
 		return -DER_INVAL;
 	}
 
+	if (ctx->ref_count > 1) {
```

Comment:
should not it always ctx->ref_count--?

Reply:
I can go that way if it's clearer. However, if it's <= 1 at this point, we want to free everything - no one will be able to check the ref count beyond that point.